statefun-flink/statefun-flink-distribution/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,40 @@
+statefun-flink-distribution
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies from (or other subprojects of) the owning top-level project, Apache Flink,
+under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- org.apache.flink:flink-annotations:1.10-SNAPSHOT
```

Comment:
`1.10-SNAPSHOT` should be updated once Flink 1.10 is released and we no longer rely on the snapshots.